bin/yamlbot

```diff
@@ -26,6 +26,10 @@ parser = OptionParser.new do |opts|
     options[:file] = file
   end
 
+  opts.on('-c', '--color', 'Enable colored output') do |color|
+    options[:color] = color
```

Comment:
Good point, I figure this is going to be ran locally by most folks within a bash terminal and the colored output would be helpful.